src/Collector/Stream/FilesystemStreamProxy.php

```diff
@@ -17,7 +17,7 @@ class FilesystemStreamProxy implements StreamWrapperInterface
      * @var resource|null
      */
     public $context;
-    public StreamWrapperInterface $decorated;
+    public StreamWrapper $decorated;
```

Comment:
For what reason?

Reply:
I assumed that `$decorated` setted in constructor only (used `StreamWrapper`).